public/request/game-relation/create.php

```diff
@@ -10,9 +10,16 @@
 
 $input = Validator::validate(Arr::wrap(request()->post()), [
     'game' => 'required|integer|exists:GameData,ID',
-    'relations' => 'required',
+    'relations' => 'required|string',
 ]);
 
-modifyGameAlternatives($user, (int) $input['game'], toAdd: $input['relations']);
+$gameId = (int) $input['game'];
+
+// Filter out instances where a game might be linked to itself.
+$relationsArray = explode(",", $input['relations']);
+$filteredArray = array_diff($relationsArray, [$input['game']]);
+$filteredRelationsCsv = implode(",", $filteredArray);
+
+modifyGameAlternatives($user, $gameId, toAdd: $filteredRelationsCsv);
```

Comment:
I think it would make more sense to pass `$filteredArray` to `modifyGameAlternatives` to avoid deconstructing the CSV again, but you'd lose the input validation (and this field is user-entered), so it's probably fine like this.